guides/doc-Release_Notes/topics/katello-contributors.adoc

```diff
@@ -0,0 +1 @@
+Adam Růžička, Bernhard Suttner, Chris Roberts, Evgeni Golov, Ewoud Kohl van Wijngaarden, Hao Chang Yu, Ian Ballou, Jeremy Lenz, Leos Stejskal, Lucy Fu, Manisha15, Maria, MariaAga, Markus Bucher, Maxim, Nagoor Shaik, Partha Aji, Patrick C. F. Ernzer, Pavel Moravec, Peter Koprda, Quinn James, Quirin Pamp, Samir Jha, Stejskal Leos, William Bradford Clark
```

Comment:
I think there may be some duplication here because I think maria and maria Aga refer to the same person. I also see `Leos Stejskal` and `Stejskal Leos`.